data/data/aws/master/variables.tf

```diff
@@ -1,8 +1,3 @@
-variable "base_domain" {
```

Comment:
Oops, looks like I added this without a consumer in 0bec46d3 (#362).  Can you pull it out into a separate PR so we can land it without waiting for this PR to settle?

Reply:
sure.